python/graphstorm/inference/utils.py

```diff
@@ -0,0 +1,89 @@
+"""
+    Copyright 2023 Contributors
```

Comment:
```suggestion
    Copyright 2025 Contributors
```

Reply:
The recommendation is to avoid years in copyright statements to avoid having to maintain them. Applicability is the same.